db/migrate/20180704190552_drop_id_from_book.rb

```diff
@@ -0,0 +1,5 @@
+class DropIdFromBook < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :books, :id
```

Comment:
Why did you do this?
WIthout ID you will not be able to "recognize" the book.
If you care about the task of the card, the ID is setted automatically by rails when you create any element